suse/sle-hpc-15.x/sle-hpc-15-sp4/fixes.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
How is this file different from install_utils.sh. Is there any reason why this is named as fixes.sh?

Reply:
Fixes are changes which are required during the lifecycle of the version.
As install_utils is move to common as its should be valid for all e.g. 15 versions we would need something in addition for a case.
Yes, it would be not necessary if install_utils would stay in the minior version tree

Reply:
I'd prefer the install_utils.sh inside of 15sp4 tree, instead of 'fixes', to be in sync with the rest of the repo